hooks/useStore.js

```diff
@@ -0,0 +1,23 @@
+import create from 'zustand';
```

Comment:
Hey Tom, everything else looks great, but I'm a bit confused by this file... why do you need the useStore/zustand at this stage? As far as I understand, you are just displaying the data you get from the json file in a card. You don't need zustand for that. Or am I missing something?

Reply:
Oh yes ur right, I deleted the folder. Thank you cintia :)